src/DebtToken.sol

```diff
@@ -28,7 +28,16 @@ abstract contract DebtToken is ERC4626 {
     // Maximum amount of `underlying asset` in debt that a single debtor can take.
     uint128 public borrowCap;
 
-    error FunctionNotImplemented();
+    /* //////////////////////////////////////////////////////////////
+                                ERRORS
+    ////////////////////////////////////////////////////////////// */
+
+    /// @notice Thrown when assets to borrow exceeds amount of debt that a single debtor can take on that asset.
```

Comment:
```suggestion
    // @notice Thrown when assets to borrow exceeds amount of debt that a single debtor can take on that asset.
```

We always use two // for inline comments in our contracts, let's be consistent

Reply:
ok done!